addons/print/models/print_printer.py

```diff
@@ -204,6 +204,7 @@ def spool_report(self, docids, report_name, data=None, title=None, copies=1):
         if printers_by_report_type:
             required_types = set(printers_by_report_type.keys())
             missing = required_types - report_types
+            test = False
```

Comment:
stray debug?

Reply:
Lol yep, while looking into another comment. 😁

Will push up the fix when I have something else to push up... :)

Reply:
Don't think I'll have anything else to push up for today so have now pushed up the commit to remove this. :)